uo_cli.gemspec

```diff
@@ -41,4 +41,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "urbanopt-scenario", "~> 0.1.0"
   spec.add_development_dependency "urbanopt-geojson", "~> 0.1.0"
+  spec.add_dependency "github_api", "~> 0.18.0"
```

Comment:
This may not be required, or desired. [OpenStudio-server](https://github.com/NREL/OpenStudio-server) uses the same `change_log.rb` script, but doesn't have a `.gemspec` file at all. Having this dependency makes the script work in its current form, though there may be a better way.

Reply:
Nick didn't want this dependency in openstudio-extension. I'm not sure how to make this work without it, or if the same concern exists in this repo.